src/subdomains/lightning/services/lightning-forward.service.ts

```diff
@@ -84,13 +89,25 @@ export class LightningForwardService {
     const amount = params.amount;
 
     if (currencyCode && amount) {
-      params.amount = await this.lightningCurrencyService.calculatePayAmount(currencyCode, amount);
+      return this.lnbitsWalletPayment(address, currencyCode, amount);
     }
 
     const lnurlpId = await this.getLnurlpId(address);
     return this.client.getLnurlpInvoice(lnurlpId, params);
   }
 
+  private async lnbitsWalletPayment(
+    address: string,
+    currencyCode: string,
+    amount: number,
+  ): Promise<LnBitsLnurlpInvoiceDto> {
+    this.lightningCurrencyService.paymentCheck(currencyCode, amount);
+
+    const adminKey = await this.getLightningWallet(address).then((lw) => lw.adminKey);
+
+    return this.client.getLnBitsWalletPayment(adminKey, amount, currencyCode).then((p) => ({ pr: p.payment_request }));
+  }

```

Comment:
Routes field is missing, see https://github.com/lnurl/luds/blob/luds/06.md
![image](https://github.com/LightningDotSpace/api/assets/85513542/ac302325-147a-4347-95fe-2065070305ac)